.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
The tags are currently not prefixed with v. We could technically change it for this repo since there is only one tag, but the main repo doesn't have a v prefix either. Seems like we can match on regex: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet. Which would be `[0-9]+.[0-9]+.[0-9]+` I guess

Reply:
I think we should move to using the prefix, it makes things easy.

I can reupdate tags to make them all use the v if that is an issue.

Reply:
I have a slight preference for the non-prefixed variant since that's what is most common among my projects and projects I've contributed to. Using the regex instead of glob pattern will ensure that we only run this for semvers (though that goes for with or without prefix).

But if you feel strongly about it remember to update all 18 releases of Lektor when we use it there.

Reply:
Sure, I have a script for doing that :-p 

Most to fthe projects I work in use the v so we are both biased :-p